src/graph/delta_matrix/delta_matrix.h

```diff
@@ -190,15 +184,17 @@ GrB_Info Delta_mxm                     // C = A * B
 	Delta_Matrix C,                    // input/output matrix for results
 	const GrB_Semiring semiring,       // defines '+' and '*' for A*B
 	const Delta_Matrix A,              // first input:  matrix A
-	const Delta_Matrix B               // second input: matrix B
+	const Delta_Matrix B,              // second input: matrix B
+	GrB_Descriptor desc                // descriptor for C=A*B
```

Comment:
I think it's a good thing we align with the GraphBLAS API, but who'd actually passing a descriptor and are we applying it to all 3 internal matrices ? `M`, `DM` and `DP` ?